docs/explanation/lex-persona.md

```diff
@@ -0,0 +1,628 @@
+            "hideWorkflowRecipients": true,
+            "id": "stp_x6WLNow6GqDRsr2k********",
+            "invitePeriod": 86400000,
+            "isFinished": true,
+            "isStarted": true,
+            "logs": [
+                {
+                    "created": 1689092760018,
+                    "operation": "start"
+                },
+                {
+                    "created": 1689092760018,
+                    "operation": "notifyWorkflowStarted"
+                },
+                {
+                    "created": 1689092760372,
+                    "operation": "invite",
+                    "recipientEmail": "<signer_email_address>@mail.com",
+                    "recipientUserId": "usr_H5RSBaPTeYu7EjsG****"
+                },
+                {
+                    "created": 1689092977709,
+                    "evidenceId": "evi_serversealing_m408RqHSvAIOAaP2********",
+                    "operation": "sign",
+                    "recipientEmail": "<signer_email_address>@mail.com",
+                    "recipientUserId": "usr_H5RSBaPTeYu7EjsG****"
+                },
+                {
+                    "created": 1689092977709,
+                    "operation": "notifyRecipientFinished",
+                    "recipientEmail": "<signer_email_address>@mail.com",
+                    "recipientUserId": "usr_H5RSBaPTeYu7EjsG********"
+                },
+                {
+                    "created": 1689092977709,
+                    "operation": "notifyWorkflowFinished"
+                }
+            ],
+            "maxInvites": 5,
+            "recipients": [
+                {
+                    "consentPageId": "cop_NzBEGbgTkj7v5tqp****",
+                    "country": "FR",
+                    "email": "<signer_email_address>@mail.com",
+                    "firstName": "John",
+                    "lastName": "Doe",
+                    "phoneNumber": "+33 7 ** ** ** **",
+                    "preferredLocale": "fr",
+                    "userId": "usr_H5RSBaPTeYu7********"
+                }
+            ],
+            "requiredRecipients": 1,
+            "sendDownloadLink": true,
+            "stepType": "signature",
+            "validityPeriod": 8553600000
+        }
+    ],
+    "tenantId": "ten_9H4TvNNQFfrLBXH2********",
+    "updated": 1689092977939,
+    "userId": "usr_P5YBCEeRqY8jKGGM********",
+    "viewAuthorizedGroups": [
+        "grp_9woWtcSt5KiAzyqL********"
+    ],
+    "viewAuthorizedUsers": [
+        "usr_H5RSBaPTeYu7EjsG********"
+    ],
+    "watchers": [],
+    "workflowStatus": "finished"
+}
+```
+
+We can see the workflowStatus has reached his final step : "finished"
+___
+#### 4- Retrieve the signed documents :
+
+For a normal signer who was invited to sign a document, he will be able to retrieve the documents through the last notification email that states the end of the worklfow.
+
+Concerning the API, we can just use the API to retrieve the documents that were signed.
```

Comment:
How long is the document stored by Lex Persona? Should we have to store it once signed on our own infrastructure or can we deal with the fact that Lex Persona document life time is long enough ?

Reply:
Lex Persona stores your document from 6 years up to 10 years. Documents can be accessible for this length of time.